.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: ruby
+install: bundle install -j 4 --retry 3
+script:
+  - RAILS_ENV=test bundle exec rspec spec
+matrix:
+  include:
+    - rvm: "1.9.3"
```

Comment:
Did you run the tests with these old versions to ensure it works? I think travis will run only in the next PRs

Reply:
@fabioperrella could you please enable Travis inside https://travis-ci.org/? By this the travis will start to work.

Reply:
In fact I've add all those ruby versions to be checked, but I don't think we need to run the tests for all versions. Maybe we can choose only 2 ruby versions to run the tests, what do you think?

Reply:
I agree!

> could you please enable Travis inside https://travis-ci.org/

done!